### DISCUSSION TITLE
EliteVA errors

### DISCUSSION THREAD

**Discussion by ZakolkaTV** [Somfic/EliteVA#16] [General]
I use the EliteVA plugin in Voice attack and write voice control. I found such a problem.

When I fly into a combat zone with targoids, EliteVA issues an error and then stops working. To remove the error, I need to delete all the elite log files. I think EliteVA is trying to check the legal status of the targoid and cannot do it. But can Targoids have a legal status? I really hope that you can help solve this problem or fix the error, if there is one.

Error from the EliteVA logs:
2024/01/26 20:45:47.372 EliteAPI.Events.Events Could not invoke event from JSON
Json serialization: Error converting value "Thargoid" to type 'EliteAPI.Events.Status.Ship.LegalState'. Path 'LegalState', line 1, position 231.
{"JSON":"{ "timestamp":"2024-01-26T17:45:44Z", "event":"Status", "Flags":20971536, "Flags2":0, "Pips":[4,4,4], "FireGroup":0, "GuiFocus":0, "Fuel":{ "FuelMain":31.987114, "FuelReservoir":0.543873 }, "Cargo":0.000000, "LegalState":"Thargoid", "Balance":6141515863 }\r\n","File":"C:\Users\aldom\Saved Games\Frontier Developments\Elite Dangerous\Status.json"}
Argument: Requested value 'Thargoid' was not found.

**Comment by Somfic** [comment]
Thanks for the report! I've tagged it as issue #17 and will try to fix it asap.

**Reply by Somfic** [reply]
Should be fixed in [v0.4.2](https://github.com/Somfic/EliteVA/releases/tag/0.4.2)!
